supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -172,25 +174,47 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
-    val referenceInputsToMountedPaths = PipelinesApiReferenceFilesMapping.getReferenceInputsToMountedPathMappings(referenceFilesMapping, inputs)
-    val referenceFilesLocalizationScript = {
-      val symlinkCreationCommands = referenceInputsToMountedPaths map {
-        case (input, absolutePathOnRefDisk) =>
-          s"mkdir -p ${input.containerPath.parent.pathAsString} && ln -s $absolutePathOnRefDisk ${input.containerPath.pathAsString}"
-      }
+    val optionName = WorkflowOptions.UseReferenceDisks.name
+    val useReferenceDisks = workflowDescriptor.workflowOptions.getBoolean(optionName) match {
+      case Success(value) => value
+      case Failure(OptionNotFoundException(_)) => false
+      case Failure(f) =>
+        // Should not happen, this case should have been screened for and fast-failed during workflow materialization.
+        log.error(f, s"Programmer error: unexpected failure attempting to read value for workflow option '$optionName' as a Boolean")
+        false
+    }
 
-      if (symlinkCreationCommands.nonEmpty) {
-        s"""
-           |# Faux-localizing reference files (if any) by creating symbolic links to the files located on the mounted reference disk
-           |${symlinkCreationCommands.mkString("\n")}
-           |""".stripMargin
-      } else {
-        ""
+    // Generate a mapping of reference inputs to their mounted paths and a section of the localization script to
+    // "faux localize" these reference inputs with symlinks to their locations on mounted reference disks.
+    def generateReferenceInputsAndLocalizationScript: (Map[PipelinesApiInput, String], String) = {
+      val referenceInputsToMountedPaths: Map[PipelinesApiInput, String] = PipelinesApiReferenceFilesMapping.getReferenceInputsToMountedPathMappings(referenceFilesMapping, inputs)
+
+      val referenceFilesLocalizationScript = {
+        val symlinkCreationCommands = referenceInputsToMountedPaths map {
+          case (input, absolutePathOnRefDisk) =>
+            s"mkdir -p ${input.containerPath.parent.pathAsString} && ln -s $absolutePathOnRefDisk ${input.containerPath.pathAsString}"
+        }
+
+        if (symlinkCreationCommands.nonEmpty) {
+          s"""
+             |# Faux-localizing reference files (if any) by creating symbolic links to the files located on the mounted reference disk
+             |${symlinkCreationCommands.mkString("\n")}
+             |""".stripMargin
+        } else {
+          "\n# No reference disks mounted / no symbolic links created since no matching reference files found in the inputs to this call.\n"
+        }
       }
+      (referenceInputsToMountedPaths, referenceFilesLocalizationScript)
+    }
+
+    val (maybeReferenceInputsToMountedPaths, maybeReferenceFilesLocalizationScript) = if (useReferenceDisks) {
```

Comment:
With a properly configured backend, this looks like it used to use reference disks by default. Now, it looks like the default is to _not_ use reference disks unless the workflow option is provided and `true`. How worried should we be about this change in default behavior?

Reply:
discussed in standup, this feature is not currently enabled in Terra prod since it is not set up in config